.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and Deploy
+on: [push]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Setup node.js 🏡
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.15.0'
```

Comment:
If we're going to test against Node 14.15, let's add an `.nvmrc` to use that locally

Reply:
This value actually came from our `.nvmrc` (that you added in b3613d1 😉 )
https://github.com/developmentseed/cdk-seed/blob/fdff42f5d5284ca7a30f343f691efa4995e06a9f/.nvmrc#L1

Reply:
Well then, touché